dev-tools/packaging/templates/windows/install-service.ps1.tmpl

```diff
@@ -1,9 +1,18 @@
 # Delete and stop the service if it already exists.
 if (Get-Service {{.BeatName}} -ErrorAction SilentlyContinue) {
-  $service = Get-WmiObject -Class Win32_Service -Filter "name='{{.BeatName}}'"
-  $service.StopService()
-  Start-Sleep -s 1
-  $service.delete()
+  try {
```

Comment:
Lets not do the try/catch and just change to the code in the catch block.

I tested the code in the catch block on Windows 11, Windows Server 2012, 2016 & 2019.  Works fine.

Reply:
I made the requested changes, please let me know if there is anything further needed.  Thanks for moving so quick on this!